packages/zero-cache/src/services/mutagen/pusher.ts

```diff
@@ -45,8 +57,28 @@ export class PusherService implements Service {
     this.id = clientGroupID;
   }
 
-  enqueuePush(push: PushBody, jwt: string | undefined) {
+  enqueuePush(
+    clientID: string,
+    wsID: string,
+    push: PushBody,
+    jwt: string | undefined,
+  ): HandlerResult {
+    const downstream: Subscription<Downstream> | undefined =
+      this.#pusher.maybeInitConnection(clientID, wsID);
```

Comment:
returns a downstream if this is the first push we've received for the given `clientID` and `wsID` combo.